gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -46,6 +84,10 @@ private Intersector () {
 	 * @param t3 the third vertex of the triangle
 	 * @return whether the point is in the triangle */
 	public static boolean isPointInTriangle (Vector3 point, Vector3 t1, Vector3 t2, Vector3 t3) {
+		final Vector3 v0 = new Vector3();
```

Comment:
Please, don't instantiate new objects inside collision methods, they are called multiple times. You can get out of memory very fast. One of the reasons we have so many static final fields is to prevent multiple instantiations.

Reply:
done! Thanks!